docs/guides/execution-modes.mdx

```diff
@@ -5,36 +5,105 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
+<video title="A user starts a session job and becomes the priority user.  They submit jobs to the QPU while other users wait. After the prioritized user's session finishes, the next person's jobs can begin processing." className="max-w-auto h-auto" controls>
+  <source src="/videos/guides/sessions/demo.mp4" type="video/mp4"></source>
+</video>
 
 <span id="sessions-versus-batch-usage" />
 
-## Choose batch or sessions mode
+## Choose the right execution mode
+
+Utility-scale workloads can take many hours to complete, so it is important that both the classical and quantum resources are scheduled efficiently to streamline the execution. Execution modes provide flexibility in balancing the cost and time tradeoff to use resources optimally for your workloads. There are several aspects to consider when choosing which execution mode to use, such as _usage_, overall execution time, ITTL requirement, and time between jobs.
+
+The following table summarizes the benefits of each:
+
+| Mode    | Benefits                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Batch   | ● The entire batch of jobs is scheduled together and there is no additional queuing time for each.<br></br>● The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.<br></br>There is usually minimal delay between jobs, which can help avoid drift.<br></br>● If you partition your workload into multiple jobs and run them in batch mode, you can get results from individual jobs, which makes them more flexible to work with. |
```

Comment:
Are the dots trying to recreate a list? We shouldn't do that because it doesn't comply with a) "semantic web" and b) Carbon design. Does it work to do this? 
```suggestion
| Batch   | <ul><li>The entire batch of jobs is scheduled together and there is no additional queuing time for each.</li><li><p>The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially.</p><p>There is usually minimal delay between jobs, which can help avoid drift.</p></li><li>If you partition your workload into multiple jobs and run them in batch mode, you can get results from individual jobs, which makes them more flexible to work with.</li> |
```

Reply:
No; the <ul> doesn't work. I have not yet figured out how to add an unordered list to a table.  The paragraphs work though, maybe that's enough.  You can look at the preview to see how it looks.

I found an example online that uses dashes (see the Job row) does that work?   @Eric-Arellano

Reply:
replaced the table with lists.